hfinger/hfinger_core.py

```diff
@@ -98,132 +93,139 @@ def get_hdr_order(request_split):
                 hdr_coded = HDRL[hdr_lower]
             else:
                 hdr_coded = "!" + HDRL[hdr_lower]
-        ret.append(hdr_coded)
-    return ",".join(ret)
+        return_list.append(hdr_coded)
+    return ",".join(return_list)
 
 
 def get_ua_value(hdr):
-    val = hdr.split(":")[1]
-    if val[0] == " ":
-        val = val[1:]
+    header_value = hdr.split(":")[1].lstrip(" ")
```

Comment:
```suggestion
    header_value = hdr.split(":", 1)[1].strip()
```
The header value may include a `:` which would break the split.